benchmarks/drbg.cc

```diff
@@ -0,0 +1,81 @@
+/*
+ *    Copyright 2022 Cryspen Sarl
+ *
+ *    Licensed under the Apache License, Version 2.0 or MIT.
+ *    - http://www.apache.org/licenses/LICENSE-2.0
+ *    - http://opensource.org/licenses/MIT
+ */
+
+#include "Hacl_HMAC_DRBG.h"
+
+#include "util.h"
+
+static bytes entropy_input = from_hex("98AD3A7FE9E908452A633AA3B9E46E7E324766");
+static bytes nonce = from_hex("6DF336AB15C2BED6AC6FC793705EBBAAFE");
+static bytes personalization_string =
+  from_hex("AF65C375F0850AF29CC3FE6C0C1D31F4");
+static bytes additional_input = from_hex("1CA8F61C");
+static bytes output(128);
+
+inline void
+Drbg_complete(benchmark::State& state, Spec_Hash_Definitions_hash_alg algorithm)
+{
+  for (auto _ : state) {
+    Hacl_HMAC_DRBG_state state = Hacl_HMAC_DRBG_create_in(algorithm);
+    Hacl_HMAC_DRBG_instantiate(algorithm,
```

Comment:
The Evercrypt is the one we really want people to use, not the HACL one. So maybe we should benchmark that one as well?
But not too important. The performance here will mostly be dominated by IO.